src/tests/class_tests/openms/source/RibonucleotideDB_test.cpp

```diff
@@ -65,7 +65,7 @@ END_SECTION
 START_SECTION(ConstIterator begin())
 {
   RibonucleotideDB::ConstIterator it = ptr->begin();
-  TEST_STRING_EQUAL((*it)->getCode(), "m1Am");
+  TEST_STRING_EQUAL((*it)->getCode(), "io6A");
 }
 END_SECTION
 
```

Comment:
can you add a test that checks if loading from tsv or json works?

Reply:
will do.

Reply:
> can you add a test that checks if loading from tsv or json works?

Looking at these tests, the loading of the TSV and JSON occurs when the constructor for RibonucleotideDB is called the first time. What's the correct way to test them outside of constructor?

Reply:
It is probably fine to test it in the test for the constructor.
But you should probably test at least one entry per file type (and add a comment what you are testing to communicate intend so it is not accidentally changed, e.g., back to only testing for entries in the TSV).